### PR TITLE
Add builder gem dependency

### PR DESCRIPTION
**Pull request by Sammers21**
`build` package is required by `Gem::Indexer` to work properly.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,6 +26,10 @@ This is the dependency you need:
 Read the [Javadoc](http://www.javadoc.io/doc/com.artipie/gem-adapter)
 for more technical details.
 
+## Extra gems
+
+`lib/lib.jar` file contains additional ruby gems. The file is generated by `ruby_lib.sh` script.
```
Comment (g4s8) [inline README.md:31]: @Sammers21 why do we need this jar? I think you need to describe it in the readme

File: pom.xml [hunk 2]
```diff
@@ -118,6 +118,13 @@ SOFTWARE.
       <artifactId>vertx-rx-java2</artifactId>
       <version>${vertx.version}</version>
     </dependency>
+    <dependency>
+      <groupId>com.artipie</groupId>
+      <artifactId>extra-gems</artifactId>
+      <version>1.0</version>
+      <scope>system</scope>
+      <systemPath>${basedir}/lib/lib.jar</systemPath>
```
Comment (g4s8) [inline pom.xml:126]: @Sammers21 how this dependency will be resolved from external sources e.g. `artipie/artipie`?

File: ruby_lib.sh [hunk 3]
```diff
@@ -0,0 +1,8 @@
+rm -rf lib
+gem fetch builder
```
Comment (g4s8) [inline ruby_lib.sh:2]: @Sammers21 why can't we include it as a ruby dependency without building jar?

**Comment by 0crat** [comment]
This pull request [#18](https://github.com/artipie/gem-adapter/issues/18) is assigned to @vatavuk[/z](https://www.0crat.com/u/vatavuk), here is [why](https://www.0crat.com/footprint/CT2E6TK9B/dc611d29-c415-4f55-8bdc-535bab77b7de); the budget is 15 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§27](http://www.zerocracy.com/policy.html#27) and when you decide to accept the changes, inform @g4s8[/z](https://www.0crat.com/u/g4s8) (the architect) right in this ticket; if you decide that this PR should not be accepted ever, also inform the architect; this [blog post](http://www.yegor256.com/2015/02/09/serious-code-reviewer.html) will help you understand what is expected from a code reviewer; there will be a monetary reward for this job

**Comment by 0crat** [comment]
@0crat[/z](https://www.0crat.com/u/0crat) I see that you unassigned this issue; the order is still assigned to @vatavuk[/z](https://www.0crat.com/u/vatavuk) though; to cancel the order use `refuse`, as in [§6](http://www.zerocracy.com/policy.html#6)

File: README.md (hunk 1 above)
Comment (Sammers21) [reply README.md:31]: the additional ruby gems required by the project

**Comment by 0crat** [comment]
@0crat[/z](https://www.0crat.com/u/0crat) I see that you unassigned this issue; the order is still assigned to @g4s8[/z](https://www.0crat.com/u/g4s8) though; to cancel the order use `refuse`, as in [§6](http://www.zerocracy.com/policy.html#6)

**Comment by 0crat** [comment]
This pull request [#18](https://github.com/artipie/gem-adapter/issues/18) is assigned to @g4s8[/z](https://www.0crat.com/u/g4s8), here is [why](https://www.0crat.com/footprint/CT2E6TK9B/0d9cab60-09af-40e5-94ee-f1abc7fa0c80); the budget is 15 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§27](http://www.zerocracy.com/policy.html#27) and when you decide to accept the changes, inform @g4s8[/z](https://www.0crat.com/u/g4s8) (the architect) right in this ticket; if you decide that this PR should not be accepted ever, also inform the architect; this [blog post](http://www.yegor256.com/2015/02/09/serious-code-reviewer.html) will help you understand what is expected from a code reviewer; there will be no monetary reward for this job

File: ruby_lib.sh (hunk 3 above)
Comment (Sammers21) [reply ruby_lib.sh:2]: What do you mean?

**Comment by Sammers21** [comment]
@g4s8 , now required ruby dependencies are packaged into a resulted jar.

File: ruby_lib.sh [hunk 4]
```diff
@@ -0,0 +1,9 @@
+set -e
+set -x
+if [ ! -d "./lib" ]
+then
+  gem fetch builder
+  BUILDER_GEM_FILE=$(ls -la | grep builder*\.gem | awk '{print $NF}')
```
Comment (g4s8) [inline ruby_lib.sh:6]: @Sammers21 it can be simplified to `ls | grep builder*\.gem`

**Comment by 0crat** [comment]
Job was finished in 20 hours, bonus for fast delivery is possible (see [§36](http://www.zerocracy.com/policy.html#36))

**Comment by 0crat** [comment]
Job audit: `ARC` didn't merge or close PR, performer @g4s8[/z](https://www.0crat.com/u/g4s8)[/z](https://www.0crat.com/u/g4s8) didn't make CR comments

**Pull request merged**